packages/workflow/src/internals.ts

```diff
@@ -306,14 +307,16 @@ export class Activator implements ActivationHandler {
             name,
             description: value.description,
           }));
-          return {
-            definition: {
-              type: workflowType,
-              queryDefinitions,
-              signalDefinitions,
-              updateDefinitions,
-            },
-          };
+          return new RawValue(
+            this.payloadConverter.toPayload({
```

Comment:
IIUC, this needs to use the _default_ payload converter, not the one provided by the user.